Header header
PointClass[] point_classes
